Containers/imaginary/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM golang:1.20.5-alpine3.17 as go
+FROM golang:1.20.5-alpine3.18 as go
 
-ENV IMAGINARY_HASH b632dae8cc321452c3f85bcae79c580b1ae1ed84
+ENV IMAGINARY_HASH b632dae8cc321452c3f85bcae79c580b1ae1ed84

```

Comment:
noidea what it did here...

Reply:
probably again line break added back. It is fine.